src/app/views/App.tsx

```diff
@@ -155,13 +155,22 @@ class App extends Component<IAppProps, IAppState> {
 
     return {
       sampleUrl: `${graphUrl}/${version}/${request}`,
-      selectedVerb: method,
+      selectedVerb: this.validateHttpMethod(method),
```

Comment:
This function can be best called from the function `getQueryStringParams()` where its usage is more uniform

Reply:
That was the plan. Thanks for mentioning it.